SR2020/2019-10-21-kit-disclaimer-form.md

```diff
@@ -0,0 +1,20 @@
+---
+to: sr2020-teams
+subject: Kit Disclaimer Forms
+attachments:
+  - https://drive.google.com/a/studentrobotics.org/file/d/1aCHnZ6o8DkLzHef1C2lz_iIkZkOjYP6P/view?usp=sharing
+---
+
+Hi all,
+
+Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we're currently working out the details for how to ship you your kit.
+
+Before we can issue a kit, you must sign a kit disclaimer form. Historically these have been signed at the event, but to speed things up, we're sending them over early, so you can get your kit quicker!
```

Comment:
The second sentence here mentions speeding things up many times over, maybe reduce that a bit?

Maybe also clarify that for teams who aren't attending Kickstart this will allow us to post out the kits sooner. If we can avoid promising a date, I think that's preferable, so maybe say something like "If you're not attending Kickstart: return your form promptly to ensure you receive your kit as soon afterwards as possible".